malcolm/modules/pmac/parts/motorpremovepart.py

```diff
@@ -0,0 +1,40 @@
+from annotypes import Anno, add_call_types
+
+from malcolm.core import PartRegistrar
+from malcolm.modules import builtin, scanning
+
+APartName = builtin.parts.APartName
+AMri = builtin.parts.AMri
+AIV = builtin.parts.AInitialVisibility
```

Comment:
We need to maintain the same name for the annotypes
```suggestion
AInitialVisibility = builtin.parts.AInitialVisibility
```